src/replication.c

```diff
@@ -651,7 +651,7 @@ void replicationFeedStreamFromPrimaryStream(char *buf, size_t buflen) {
     /* Debugging: this is handy to see the stream sent from primary
      * to replicas. Disabled with if(0). */
     if (0) {
-        if (server.hide_user_data_from_log) {
+        if (!server.hide_user_data_from_log) {
```

Comment:
I just happened to be debugging it and found that. Let me know if you think we should make it become a serverLog.

Reply:
I think your code is correct. We should avoid to print these information if hide_user_data_from_log flag is 1.

Reply:
IMO, I would just merge the PR as is.